website/views/options/settings.jade

```diff
@@ -397,7 +397,7 @@ script(id='partials/options.settings.subscription.html',type='text/ng-template')
         div(ng-if='!user.purchased.plan.customerId || (user.purchased.plan.customerId && user.purchased.plan.dateTerminated)')
           h4(ng-if='(user.purchased.plan.customerId && user.purchased.plan.dateTerminated)')= env.t("resubscribe")
           .form-group.reduce-top-margin
-            .radio(ng-repeat='block in Content.subscriptionBlocks | toArray | omit: "discount==true" | orderBy:"months"')
+            .radio(ng-repeat='block in Content.subscriptionBlocks | toArray | omit: "discount==true" | orderBy:"months"', ng-if="block.key !== 'group_monthly'")
```

Comment:
we could use something like `block.key.indexOf('group-') === -1` so that we don't have update this every time a new group plan is added